demisto_sdk/commands/common/hook_validations/release_notes.py

```diff
@@ -536,5 +536,9 @@ def is_file_valid(self):
             self.validate_json_when_breaking_changes(),
             # self.has_no_markdown_lint_errors(),
             self.validate_release_notes_headers(),
+            self.validate_no_disallowed_terms_in_customer_facing_docs(
+                file_content=self.latest_release_notes,
+                file_path=self.release_notes_file_path,
```

Comment:
Are we sure we want to send the file content if we are already sending the path?

Reply:
The validation actually runs on the content, `file_path` is just used for the error handler (to print which file failed the validation).

I think it'll be bad optimization-wise to reopen & load it to a string again within the validation function.